spec/components/guardian_spec.rb

```diff
@@ -539,15 +550,26 @@
       let(:group_owner) { group_private_topic.user.tap { |u| group.add_owner(u) } }
       fab!(:pm) { Fabricate(:topic) }
 
+      it 'returns true if user trust level matches pm invite site setting' do
+        SiteSetting.min_trust_to_allow_pm_invite = 2
```

Comment:
I don't think we need this line since it has already been set in the `before` block.